extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui/controller/jsonrpc.js

```diff
@@ -205,7 +205,7 @@ export class JsonRpc {
 
     _setExtensionName(discoveredNamespace, serviceIdentifier){
         if(serviceIdentifier){
-            this._extensionName = discoveredNamespace + "-" + serviceIdentifier;
+            this._extensionName = serviceIdentifier;
```

Comment:
@phillip-kruger I need your opinion on this change: I had a look and nobody is using the `serviceIdentifier` as far as I can tell. My use case is to be able to use the same service for different pages (so different `discoveredNamespace`).

I have the `build-metrics` service and I want to use it in both `qwc-build-steps` and `qwc-build-items`.

Maybe there's another way to achieve that and this was actually useful for another use case.

Reply:
This will be used in Kafka UI (@cescoffier needs it). So the idea is that you can have multiple Json RPC backends in one extension (so that you can separate logic). In your case just omitting the name everywhere should work, since both of the components run under the same namespace. You can  however, in advance cases where you know what the namespace is do what you want here, without the code change you suggest. You can do `jsonRpc = new JsonRpc('build-metrics', true);` So this is more when you want to use the JsonRPC services of another component. Example @melloware wanted to use the server log from the Quinoa extension, so this allows you to do that.

Reply:
@phillip-kruger unfortunately it didn't work. IIRC, I ended up with `build-steps-build-metrics` and `build-items-build-metrics`: the namespace wasn't the same.

Reply:
Ok, we still can not make that change using the service name as is, I did a commit that solve this for now, I am busy with some refactoring in my current PR for Continuous testing that will fix this better. For now please look at my commit